### PR TITLE
RDK-35868: Enhancing the L1_testing

### PR DESCRIPTION
**Pull request by chandana0902**
enhancing the L1 testing to support few API which was not covered

### CODE REVIEW

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py [hunk 1]
```diff
@@ -1402,5 +1400,1381 @@ def test_optional_feild_app_meta_data_schema(self):
         self.assertEqual(actual, expected)
         logger.debug("-->Test was Successfully verified")
 
+    def test_network_feild_compatilibity(self):
```
Comment (goruklu) [inline unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py:1403]: typo
`feild_compatilibility` should be `field_compatibility`

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py [hunk 2]
```diff
@@ -1402,5 +1400,1381 @@ def test_optional_feild_app_meta_data_schema(self):
...
+            },
+            "rdk": {
+                "supportedFeatures": []
+            }
+        }
+        actual = processor.check_compatibility()
+        expected = False
+        self.assertEqual(actual, expected)
+
+    def test_graphic_feild_compatilibity(self):
```
Comment (goruklu) [inline unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py:1433]: again same typo

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py [hunk 3]
```diff
@@ -1402,5 +1400,1381 @@ def test_optional_feild_app_meta_data_schema(self):
...
+        }
+        processor.platform_cfg = {
+            "hardware":{
+            }
+        }
+        actual = processor.check_compatibility()
+        expected = False
+        self.assertEqual(actual, expected)
+
+    def test_persistent_storage_feild_compatilibity(self):
```
Comment (goruklu) [inline unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py:1449]: typo

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py [hunk 4]
```diff
@@ -1402,5 +1400,1381 @@ def test_optional_feild_app_meta_data_schema(self):
...
+            "rdk":{
+            },
+            "storage":{
+            }
+        }
+        actual = processor.check_compatibility()
+        expected = False
+        self.assertEqual(actual, expected)
+
+    def test_storage_persistent_maxsize_feild_compatilibity(self):
```
Comment (goruklu) [inline unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py:1475]: typo

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py [hunk 5]
```diff
@@ -1402,5 +1400,1381 @@ def test_optional_feild_app_meta_data_schema(self):
...
+            "mounts":[],
+            'rdkPlugins': {
+                'storage': {
+                    'data': {
+                        'loopback': [{
+                            'destination': '/home/private',
+                                'flags': 14,
+                                'fstype': 'ext4',
+                                'imgsize': 10485760,
+                                'source': '/opt/dac_apps/data/0/dac/com.rdk.flutter/23e959658f5082a89c6db72c842271a117887b8658ab703f60eaba650b3d5f20.img'}]
```
Comment (goruklu) [inline unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py:2358]: where does the `23e959658f5082a89c6db72c842271a117887b8658ab703f60eaba650b3d5f20.img` name come from ? is it hard coded ?

**Review by goruklu — CHANGES_REQUESTED** [review]

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py (hunk 1 above)
Comment (chandana0902) [reply unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py]: thank you, I will correct and update

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py (hunk 2 above)
Comment (chandana0902) [reply unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py]: Thanks,will update

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py (hunk 3 above)
Comment (chandana0902) [reply unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py]: Thanks, will update

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py (hunk 4 above)
Comment (chandana0902) [reply unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py]: Thanks will update

File: unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py (hunk 5 above)
Comment (chandana0902) [reply unit_tests/L1_testing/test_files/test_Bundleprocessor_ut.py:2358]: No, it's not hard coded. .img in the source path is formed from the hash_key of dest_path.
https://github.com/rdkcentral/BundleGen/blob/0d282784b9f3d81e766416a8d6ccd20ebf8154fc/bundlegen/core/bundle_processor.py#L861

**Review by goruklu — APPROVED** [review]

**Pull request merged**